scripts/beesd.conf.sample

```diff
@@ -15,8 +15,8 @@ UUID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
 # BEESHOME="$MNT_DIR/.beeshome"
 # BEESSTATUS="$WORK_DIR/$UUID.status"
 
-## Default options to apply, see --help for details
-# OPTIONS="--relative-paths --notimestamps"
+## Options to apply, see `beesd --help` for details
+# OPTIONS="--strip-paths --no-timestamps"
```

Comment:
Generally config files display the default options in comments (and bees dues with the other variables in this config file).

Perhaps instead of "--strip-paths and --no-timestamps" this should use "--absolute-paths --timestamps".